config/appstudio-pipelines-runner/base/appstudio_pipelines_runner_role.yaml

```
@@ -25,3 +25,12 @@ rules:
   verbs:
   - get
   - list
+- apiGroups:
+  - toolchain.dev.openshift.com
+  resources:
+  - spacerequests
+  verbs:
+  - get
+  - create
+  - delete
```

do you need also the other verbs ? 

```suggestion
  - get
  - create
  - delete
  - patch
  - list
  - update
  - watch
```

---

So you don't need to watch or update the SpaceRequest once created ?